news/2019-01-10-community-mentorship-program-spring-2019-signups-now-open.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "Community Mentorship Program Spring 2019 Signups Now Open"
+date: 2019-01-10 12:00:00 +0000
+---
+Ever wanted to get into beatmapping for this game, but have no clue where to start at? In that case, the Community Mentorship program might be the way to start your mapping career!
+
+![](/wiki/shared/news/2019-01-10-community-mentorship-program-spring-2019-signups-now-open/banner.jpg)
+
+This program serves as a platform for new mappers and modders to get into the process more quickly by teaming them up with a more experienced mapper and modder in the community to show them the ropes for a total of 10 weeks.
```

Comment:
```suggestion
This program serves as a platform for new mappers and modders to get into the beatmapping process more quickly. They'll be teaming up with more experienced mappers and modders in the community that will show them the ropes for a total of ten weeks.
```
it's a bit of a run-on, and 10->ten since no comparison is being made